addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/KeypadHandler.java

```diff
@@ -8,223 +8,183 @@
+                ledList.addAll(Arrays.asList(Component.LED1, Component.LED2, Component.LED3, Component.LED5,
+                        Component.LED6, Component.LED7));
+                break;
+            case "W3BRL":
+                buttonList.addAll(Arrays.asList(Component.BUTTON2, Component.BUTTON3, Component.BUTTON4));
+                buttonList.addAll(Arrays.asList(Component.LOWER2, Component.RAISE2));
+                ledList.addAll(Arrays.asList(Component.LED2, Component.LED3, Component.LED4));
+                break;
+            case "W3BSRL":
+            case "H3BSRL":
+                buttonList.addAll(Arrays.asList(Component.BUTTON1, Component.BUTTON3, Component.BUTTON5));
+                buttonList.addAll(Arrays.asList(Component.LOWER2, Component.RAISE2));
+                ledList.addAll(Arrays.asList(Component.LED1, Component.LED3, Component.LED5));
+                break;
+            case "W4S":
+            case "H4S":
+                buttonList.addAll(Arrays.asList(Component.BUTTON1, Component.BUTTON2, Component.BUTTON3,
+                        Component.BUTTON4, Component.BUTTON6));
+                buttonList.addAll(Arrays.asList(Component.LOWER2, Component.RAISE2));
+                ledList.addAll(
+                        Arrays.asList(Component.LED1, Component.LED2, Component.LED3, Component.LED4, Component.LED6));
+                break;
+            case "W5BRL":
+            case "H5BRL":
+            case "W5BRLIR":
+                buttonList.addAll(Arrays.asList(Component.BUTTON1, Component.BUTTON2, Component.BUTTON3,
+                        Component.BUTTON4, Component.BUTTON5));
+                buttonList.addAll(Arrays.asList(Component.LOWER2, Component.RAISE2));
+                ledList.addAll(
+                        Arrays.asList(Component.LED1, Component.LED2, Component.LED3, Component.LED4, Component.LED5));
+                break;
+            case "W6BRL":
+            case "H6BRL":
+                buttonList.addAll(Arrays.asList(Component.BUTTON1, Component.BUTTON2, Component.BUTTON3,
+                        Component.BUTTON4, Component.BUTTON5, Component.BUTTON6));
+                buttonList.addAll(Arrays.asList(Component.LOWER2, Component.RAISE2));
+                ledList.addAll(Arrays.asList(Component.LED1, Component.LED2, Component.LED3, Component.LED4,
+                        Component.LED5, Component.LED6));
+                break;
+            case "W7B":
+                buttonList.addAll(Arrays.asList(Component.BUTTON1, Component.BUTTON2, Component.BUTTON3,
+                        Component.BUTTON4, Component.BUTTON5, Component.BUTTON6, Component.BUTTON7));
+                ledList.addAll(Arrays.asList(Component.LED1, Component.LED2, Component.LED3, Component.LED4,
+                        Component.LED5, Component.LED6, Component.LED7));
+                break;
+            default:
+                logger.warn("No valid keypad model defined ({}). Assuming Generic model.", mod);
```

Comment:
If this is an optional configuration parameter it is by design so it should be logged on debug instead.